public/images/icons/addlroles.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="35" height="27" viewBox="0 0 35 27" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M32.3922 8.13655H15.2336L18.0209 4.27422C18.3837 3.77154 18.5621 3.15921 18.5263 2.54034C18.4904 1.92146 18.2424 1.33385 17.824 0.876444C17.4056 0.419037 16.8423 0.119772 16.2291 0.0290412C15.6158 -0.0616895 14.9901 0.0616554 14.4571 0.378308L5.67773 5.59475C4.14399 6.48231 2.83823 7.71513 1.86405 9.19538C0.889879 10.6756 0.274111 12.3625 0.0656132 14.1223C-0.104345 15.6851 0.0567808 17.2663 0.5385 18.7627C1.02022 20.2592 1.81173 21.6374 2.86149 22.8076C3.91124 23.9778 5.19571 24.9138 6.63125 25.5546C8.06679 26.1955 9.6212 26.5267 11.1933 26.5269C11.1933 26.5269 15.4795 26.5289 18.3627 26.5289C19.9601 26.5289 20.9716 25.5456 20.9716 24.3314V24.2764C20.9739 23.711 20.7516 23.1679 20.3536 22.7665C19.9555 22.365 19.4144 22.1381 18.849 22.1355H20.9004C21.1802 22.1355 21.4572 22.0804 21.7156 21.9734C21.9741 21.8663 22.2089 21.7094 22.4068 21.5116C22.6046 21.3138 22.7615 21.0789 22.8685 20.8205C22.9756 20.562 23.0307 20.285 23.0307 20.0053V19.9503C23.0319 19.6612 22.976 19.3746 22.8662 19.1071C22.7564 18.8396 22.5949 18.5964 22.3909 18.3914C22.187 18.1864 21.9446 18.0237 21.6776 17.9125C21.4107 17.8014 21.1244 17.744 20.8353 17.7437L22.8022 17.7429C23.3591 17.7519 23.8997 17.5547 24.3201 17.1894C24.7405 16.824 25.011 16.3161 25.0796 15.7633C25.1097 15.4584 25.0757 15.1505 24.9797 14.8596C24.8837 14.5686 24.7278 14.3009 24.5221 14.0738C24.3164 13.8467 24.0654 13.6652 23.7853 13.5409C23.5052 13.4166 23.2022 13.3524 22.8958 13.3522L32.3923 13.352C32.7348 13.352 33.0739 13.2845 33.3903 13.1534C33.7066 13.0224 33.9941 12.8303 34.2363 12.5881C34.4784 12.346 34.6705 12.0585 34.8015 11.7421C34.9326 11.4258 35 11.0867 35 10.7442C35 10.4018 34.9325 10.0627 34.8015 9.74628C34.6704 9.4299 34.4783 9.14243 34.2362 8.90029C33.994 8.65815 33.7066 8.46607 33.3902 8.33502C33.0738 8.20398 32.7347 8.13654 32.3922 8.13654V8.13655Z" fill="#3E54A1"/>
```

Comment:
we've been data urls embedded in the css to minimize the number of requests we make. Can you move to icons.data.svg.css?

Reply:
k, I'll give that a go